index.html

```diff
@@ -110,72 +112,182 @@
             };
         }
 
-		$(document).ready(function(){
-			
+        /* Set up TinCanJS */
+        var tincan = new TinCan (
+            {
+                recordStores: [
+                    {
+                        endpoint: Config.endpoint,
+                        username: Config.authUser,
+                        password: Config.authPassword,
+                        allowFail: false
+                    }
+                ]
+            }
+        ),
+        PrototypesLauncher = {};
+        PrototypesLauncher.Activity = {
+            id: "http://id.tincanapi.com/activity/tincan-prototypes/launcher",
+            definition: {
+                type: "http://id.tincanapi.com/activitytype/lms",
+                name: {
+                    "en-US": "Tin Can Prototypes Launcher"
+                },
+                description: {
+                    "en-US": "A tool for launching the Tin Can prototypes. Simulates the role of an LMS in launching experiences."
+                }
+            }
+        };
+        PrototypesLauncher.context = {
+            contextActivities : {
+                grouping: [
+                    {
+                        id: "http://id.tincanapi.com/activity/tincan-prototypes"
+                    }
+                ],
+                category: [
+                     {
+                        id: "http://id.tincanapi.com/recipe/tincan-prototypes/launcher/1",
+                        definition: {
+                            type: "http://id.tincanapi.com/activitytype/recipe"
+                        }
+                    },
+                    {
+                        id: "http://id.tincanapi.com/activity/tincan-prototypes/launcher-template",
+                        definition: {
+                            type: "http://id.tincanapi.com/activitytype/source",
+                            name: {
+                                "en-US": "Tin Can Launcher Template"
+                            },
+                            description: {
+                                "en-US": "A launch tool based on the Tin Can launch prototypes."
+                            }
+                        }
+                    }
+                ]
+            }
+        };
+
+        tincan.sendStatement({  
```

Comment:
Will you pick this up in your fixes PR or add to the list to do after this is merged? I'll need some direction if it goes on the post PR list.